examples/cull-idle/Cull_service_UI/Readme.md

```diff
@@ -0,0 +1,31 @@
+<h2><strong>Cull Service UI Monitoring</strong></h2>
```

Comment:
@minrk I worte the html syntax in the markdown file as github renders it properly in the readme page, I can change it to markdown syntax if you suggest.